### ISSUE TITLE
Image aspect ratios not maintained for portable infobox galleries

### ISSUE THREAD

**Issue by puxlit** [Wikia/mobile-wiki#775]
For portable infobox galleries, the [current mobile styles](https://github.com/Wikia/mobile-wiki/blob/release-209.001/app/styles/module/article/_portable-infobox.scss) set `img.article-media-placeholder` to `width: 100%` within a `.pi-item` container with `width: 100vw`. This effectively distorts the images' aspect ratios, which is rather unseemly.

An example from [[[w:c:bokunoheroacademia:Yosetsu Awase]]](https://bokunoheroacademia.wikia.com/wiki/Yosetsu_Awase) follows.

![Screenshot](https://user-images.githubusercontent.com/1405147/41810481-8b72ebea-7742-11e8-900f-5c016b159c88.png)

**Comment by puxlit** [comment]
Here's one possible fix.

```diff
diff --git a/app/styles/module/article/_portable-infobox.scss b/app/styles/module/article/_portable-infobox.scss
index 99ece6166..bcd45c049 100644
--- a/app/styles/module/article/_portable-infobox.scss
+++ b/app/styles/module/article/_portable-infobox.scss
@@ -112,6 +112,7 @@ $group-header-font-size: 16px;
                .article-media-placeholder {
                        height: auto;
                        max-height: 100vw;
+                       object-fit: contain;
                        width: 100%;
                }
 
```

An example from [[[w:c:bokunoheroacademia:Yosetsu Awase]]](https://bokunoheroacademia.wikia.com/wiki/Yosetsu_Awase) with that patch applied follows.

![Screenshot](https://user-images.githubusercontent.com/1405147/41821237-e2968358-7820-11e8-8a93-cb4611a2dbdd.png)

**Comment by puxlit** [comment]
Fix merged in #821 for [release-223.001](https://github.com/Wikia/mobile-wiki/tree/92c72cd839af4be1fe5b6e292fd2463f9069f064).
